### PR TITLE
RES-1695 Sentry

### PR DESCRIPTION
**Pull request by edwh**
Remove dead code:
* CachingRssRetriever / CachingWikiPageRetriever / Rss
* Couple of unused methods from LcaStats
* TestController and associated blades/routes.
* Post-registration welcome email (turned off via feature switch, agreed to drop with James)
* PartController::stats has unused parameter.

Test and fix:
* AdminController getStats methods.
* PartyController individual group events, geocode failure.
* AcceptUserInvites middleware.
* LangaugeSwitcher middleware.
* Use e() rather than htmlspecialchars() for better escaping (e.g. quotes).

### CODE REVIEW

File: app/Http/Middleware/LanguageSwitcher.php [hunk 1]
```diff
@@ -27,9 +27,9 @@ public function handle($request, Closure $next)
             if (in_array($locale, LaravelLocalization::getSupportedLanguagesKeys())) {
                 $this->setLocale($locale);
             }
-        } elseif (session('locale')) {
+        } elseif (session('locale') && session('locale') != 'UT') {
```
[the remark this answers is not in the document]
Comment (restart-tech) [reply app/Http/Middleware/LanguageSwitcher.php:30]: Oh, it's specifically for unit tests?

**Review by restart-tech — COMMENTED** [review]
Could you double check with Monique regarding the TestController and associated routes - given the name, I imagine she'll be fine with it, but worth a check.  If we want to keep it, we should change the name to something else - e.g. UtilsController, etc.

File: app/Http/Middleware/LanguageSwitcher.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (edwh) [reply app/Http/Middleware/LanguageSwitcher.php:31]: Done.

**Comment by edwh** [comment]
I was too hasty in removing TestController; reinstated but renamed to reflect its style use.

**Pull request merged**